README.md

```diff
@@ -1,707 +1,1799 @@
 # Reselect
 
+[![npm package][npm-badge]][npm][![Coveralls][coveralls-badge]][coveralls][![GitHub Workflow Status][build-badge]][build]![TypeScript][typescript-badge]
+
 A library for creating memoized "selector" functions. Commonly used with Redux, but usable with any plain JS immutable data as well.
 
-- Selectors can compute derived data, allowing Redux to store the minimal possible state.
-- Selectors are efficient. A selector is not recomputed unless one of its arguments changes.
-- Selectors are composable. They can be used as input to other selectors.
+- Selectors can compute derived data, allowing `Redux` to store the minimal possible state.
```

Comment:
:hand: style note: we typically leave the name of the library as a proper capitalized name ("Reselect", "Redux", "React-Redux" "Redux Toolkit"), and specifically use backticks for the lower-case actual package name (`reselect`, `redux`, `react-redux`, `@reduxjs/toolkit`).  Could you fix the names?

Reply:
Yes sir, sorry about that. I will fix that right away!